### DISCUSSION TITLE
http://www.questpdf.com/ down?

### DISCUSSION THREAD

**Discussion by JakeAlmer** [QuestPDF/QuestPDF#526] [General]
Any one else having trouble getting http://www.questpdf.com/ to load?

Pretty sure its not just me:
https://downforeveryoneorjustme.com/questpdf.com?proto=http&www=1

**Comment by qcz** [comment]
Yes, down for me too.

**Comment by hahyes** [comment]
Yep, still down. Can't look into docs when I need it. :/

**Reply by JakeAlmer** [reply]
You can use https://web.archive.org/web/20220813100110/https://www.questpdf.com/introduction.html in the mean time

**Comment by MarcinZiabek** [comment]
The documentation webpage should be up again. I am still not completely sure if recovered everything properly. Honestly, DNS configuration with all firewalls, certificates, CDNs, etc. are a black magic for me. 

Suffice to say, I wasn't expecting such a high bill for traffic overage for last three months 🤣
